src/js/player.js

```diff
@@ -516,14 +532,14 @@ class Player extends Component {
       height = this.height_;
     } else {
       // Otherwise calculate the height from the ratio and the width
-      height = width  * ratioMultiplier;
+      height = width * ratioMultiplier;
     }
 
     // Ensure the CSS class is valid by starting with an alpha character
-    if (/^[^a-zA-Z]/.test(this.id())) {
-      idClass = 'dimensions-'+this.id();
+    if ((/^[^a-zA-Z]/).test(this.id())) {
+      idClass = `dimensions-${this.id()}`;
```

Comment:
did the linter complain about this? The reason we kept it the other way is because using the template strings here gives us no benefits over just string concatenation, and if anything, the old way is a bit cleaner.